examples/step-17/step-17.cc

```diff
@@ -359,48 +416,55 @@ namespace Step17
 
   // @sect4{ElasticProblem::assemble_system}
 
-  // We now assemble the matrix and right hand side of
-  // the problem. There are some things worth mentioning before we go into
-  // detail. First, we will be assembling the system in parallel, i.e. each
-  // process will be responsible for assembling on cells that belong to this
-  // particular processor. Note that the degrees of freedom are split in a way
-  // such that all DoFs in the interior of cells and between cells belonging
-  // to the same subdomain belong to the process that <code>owns</code> the
-  // cell. However, even then we sometimes need to assemble on a cell with a
-  // neighbor that belongs to a different process, and in these cases when we
-  // write the local contributions into the global matrix or right hand side
-  // vector, we actually have to transfer these entries to the other
-  // process. Fortunately, we don't have to do this by hand, PETSc does all
-  // this for us by caching these elements locally, and sending them to the
-  // other processes as necessary when we call the <code>compress()</code>
-  // functions on the matrix and vector at the end of this function.
+  // We now assemble the matrix and right hand side of the
+  // problem. There are some things worth mentioning before we go into
+  // detail. First, we will be assembling the system in parallel,
+  // i.e., each process will be responsible for assembling on cells
+  // that belong to this particular process. Note that the degrees of
+  // freedom are split in a way such that all DoFs in the interior of
+  // cells and between cells belonging to the same subdomain belong to
+  // the process that <code>owns</code> the cell. However, even then
+  // we sometimes need to assemble on a cell with a neighbor that
+  // belongs to a different process, and in these cases when we add up
+  // the local contributions into the global matrix or right hand side
+  // vector, we have to transfer these entries to the process that
+  // owns these elements. Fortunately, we don't have to do this by
+  // hand, PETSc does all this for us by caching these elements
```

Comment:
This is not particularly important, but I believe this sentence should use a semicolon before PETSc and not a comma.